tests/Unit/Domain/Test_InterfaceItems.cpp

```diff
@@ -105,28 +113,37 @@ struct NegateDoubleAddInt : db::ComputeTag {
 };
 
 struct AddThree : db::ComputeTag, Int {
```

Comment:
should this be `AddThreeCompute` and derived off of `AddThree` (not `Int`) ?

Reply:
I'll take the easier path of AddThree -> IntCompute